plugin/pkg/admission/security/podsecuritypolicy/admission.go

```diff
@@ -256,15 +258,19 @@ func (c *PodSecurityPolicyPlugin) computeSecurityContext(a admission.Attributes,
 	)
 
 	for _, provider := range providers {
-		podCopy := pod.DeepCopy()
+		externalPod := &corev1.Pod{}
+		if err := k8s_api_v1.Convert_core_Pod_To_v1_Pod(pod, externalPod, nil); err != nil {
```

Comment:
using the external PSP objects makes sense (because the admission plugin is acting like an API client, watching those policy objects via an informer), but why are we changing the admission interfaces and utilities to work with external v1 pod/container objects, and then doing conversions here? I expected the pod/container references to remain internal. 

cc @deads2k

Reply:
how about now? ptal

Reply:
externalPod is unused now, right? can we drop this change?